packages/calcite-components/.storybook/main.ts

```diff
@@ -23,4 +23,42 @@ module.exports = {
       ],
     };
   },
+  managerHead: (head: string): string => {
+    if (process.env.STORYBOOK_SCREENSHOT_TEST_BUILD) {
+      return head;
+    }
+
+    return `
+      <link rel="stylesheet" href="https://webapps-cdn.esri.com/CDN/fonts/v1.4.1/fonts.css" />
+      <link rel="stylesheet" href="./build/calcite.css" />
+      <script type="module" src="./build/calcite.esm.js"></script>
+
+      <template id="internalStorybookNotice">
+        <calcite-notice open icon="exclamation-mark-triangle" closable kind="warning" scale="l" style="font-family: var(--calcite-sans-family)">
+          <div slot="title">This storybook is meant for internal, testing purposes only (using @next)</div>
```

Comment:
Nitpick:

 Maybe something like this would be more clear that it is using `next` itself and not just a tool for testing `next` in some way.

> This storybook is on the current `@next` version and is meant for internal, testing purposes only.